client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,48 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakup.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakup = ( event ) => {
+	if ( ! event?.fee_rates?.history ) {
+		return [];
+	}
+
+	const {
+		fee_rates: { history },
+	} = event;
+
+	const feeLabelMapping = {
+		base: __( '\u00A0 • Base fee %s%%', 'woocommerce-payments' ),
+		'additional-international': __(
+			'\u00A0 • International fee %s%%',
+			'woocommerce-payments'
+		),
+		'additional-fx': __(
+			'\u00A0 • Foreign exchange fee %s%%',
+			'woocommerce-payments'
+		),
+	};
+
+	return history.map( ( fee ) => {
+		let labelKey = fee.type;
+		if ( fee.additional_type ) {
+			labelKey += `-${ fee.additional_type }`;
+		}
+
+		const { percentage_rate: percentageRate } = fee;
```

Comment:
Why are we only using the percentage rate here when the historical fee object also has a fixed fee? I think we can omit it when zero, but it should be included otherwise.

For example, the breakdown currently displays `Base fee 1.4%`, but skips the `£0.10` fixed fee. I know that most fee structures only apply to the percentage fee currently, but the solution would be more future-proof if we were to display both.

Reply:
Makes sense, fixed it in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/25fc362ff7b03b354fab7b7117e6aeb4ef15ccce